Source/WEB_ROOT/Inventory/js/cookies.js

```diff
@@ -0,0 +1,31 @@
+function setCookie(cname, cvalue, exdays) {

```

Comment:
Nice job utilizing single responsibility here. The cookies.js is focused on managing cookies and nothing else.